core/assets/bundles/bundle_oh_NO.properties

```diff
@@ -0,0 +1,1136 @@
+tutorial.pause = [scarlet]oh no,[accent] oh no[].\n\n[accent]Space to oh no.
+tutorial.pause.mobile = [scarlet]oh no,[accent] oh no[].\n\n[accent]oh no to oh no.
+tutorial.unpause = [scarlet]Space again to oh no.
+tutorial.unpause.mobile = [scarlet]oh no to oh no.
+tutorial.breaking = [scarlet]oh no needs to oh no\n[accent]oh no MOUSE2[] to oh no[]\n\n[accent]oh no all the oh no to the oh no using oh no.
+tutorial.breaking.mobile = [scarlet]oh no needs to oh no\n[accent]oh no[], then oh no.\noh no[] and oh no.\noh no the oh no to oh no.\n\n[accent]oh no all the oh no to the oh no using oh no.
+tutorial.withdraw = [scarlet]oh no.\nTo oh no, [accent]oh no an oh no[] oh no in it, then [accent]oh no the oh no[] in the oh no.\nMultiple oh nos can oh no via [accent]oh no[].\n\n[accent]oh no some oh no from the oh no.[]
+tutorial.deposit = [scarlet]oh no into oh no by oh no from your oh no to an oh no.\n\n[accent]oh no your oh no back to the oh no[]
+tutorial.waves = [scarlet][lightgray] oh no[] approaches.\n\noh no for 2 oh nos.[accent] MOUSE1[] to oh no.\noh no more oh no and oh no. oh no more oh no.
+tutorial.waves.mobile = [scarlet][lightgray] oh no[] approaches.\n\noh no for 2 oh nos. Your oh no will oh no the oh nos.\noh no more oh no and oh no. oh no more oh no.
+tutorial.launch = [scarlet]Once you oh no, you are able to[accent] oh no[], oh noing your oh no and[accent] oh no from your oh no[]\noh no can be oh noed to oh no.\n\n[accent]oh no the oh no.
+
+item.copper.description = [scarlet]oh no
+item.lead.description = [scarlet]oh no
+item.metaglass.description = [scarlet]oh no
+item.graphite.description = [scarlet]oh no
+item.sand.description = [scarlet]oh no
+item.coal.description = [scarlet]oh no
+item.titanium.description = [scarlet]oh no
+item.thorium.description = [scarlet]oh no
+item.scrap.description = [scarlet]oh no
+item.silicon.description = [scarlet]oh no
+item.plastanium.description = [scarlet]oh no
+item.phase-fabric.description = [scarlet]oh no
+item.surge-alloy.description = [scarlet]oh no
+item.spore-pod.description = [scarlet]oh no
+item.blast-compound.description = [scarlet]oh no
+item.pyratite.description = [scarlet]oh no
+liquid.water.description = [scarlet]oh no
+liquid.slag.description = [scarlet]oh no
+liquid.oil.description = [scarlet]oh no
+liquid.cryofluid.description = [scarlet]oh no
+mech.alpha-mech.description = [scarlet]oh no
+mech.delta-mech.description = [scarlet]oh no
+mech.tau-mech.description = [scarlet]oh no
+mech.omega-mech.description = [scarlet]oh no
+mech.dart-ship.description = [scarlet]oh no
+mech.javelin-ship.description = [scarlet]oh no
+mech.trident-ship.description = [scarlet]oh no
+mech.glaive-ship.description = [scarlet]oh no
+unit.draug.description = [scarlet]oh no
+unit.spirit.description = [scarlet]oh no
+unit.phantom.description = An advanced drone unit. Follows users. Assists in block construction.
```

Comment:
That's not an oh no.

Reply:
oh no

Reply:
I need to update the bundle to include steam text anyways